mbofui/app/index.html

```diff
@@ -115,6 +115,20 @@ <h4 class="modal-title" id="bofModalLabel">{{strings.modal_title}}</h4>
             <span class="req-text">(required)</span><span class="endToSoon alert-inline" style="display:none">&nbsp;&nbsp;Cannot end events in the past.</span>
             <input id="endTime" name="endTime" min="" class="form-control" type="datetime-local">
           </div>
+          <div class="form-group hashTagAndAddess">
```

Comment:
Minor typo:  You left the "r" out of "Address" in "hashTagAndAddess".

And FWIW, I wouldn't capitalize the "t" of "hashtag", either.